ui/components/NotificationCenter/metadata.js

```diff
@@ -1,444 +1,44 @@
-        const isMultipleRelationships =
-          Array.isArray(detail.Relationships) && detail.Relationships.length > 1;
-        const hasComponents = Array.isArray(detail.Components) && detail.Components.length > 0;
-        const hasRelationships =
-          Array.isArray(detail.Relationships) && detail.Relationships.length > 0;
-        const hasErrors = Array.isArray(detail.Errors) && detail.Errors.length > 0;
-
-        return (
-          <Box key={index} mb={2}>
-            <Typography gutterBottom>
-              <span style={{ fontWeight: 'bold', fontSize: '17px' }}>
-                {isEntityFile ? 'FILE NAME:' : 'MODEL:'}{' '}
-              </span>
-              <span style={{ fontSize: '18px' }}>{modelName}</span>
-            </Typography>
-            {hasComponents && (
-              <>
-                <Typography variant="body1">
-                  <span style={{ fontWeight: 'bold', fontSize: '14px', marginLeft: '1rem' }}>
-                    {isMultipleComponents ? 'COMPONENTS:' : 'COMPONENT:'}
-                  </span>
-                </Typography>
-                {detail.Components.map((component, idx) => (
-                  <Box key={idx} ml={2}>
-                    <ComponentWithIcon component={component} />
-                  </Box>
-                ))}
-              </>
-            )}
-            {hasRelationships && (
-              <>
-                <Typography variant="body1">
-                  <span style={{ fontWeight: 'bold', fontSize: '14px', marginLeft: '1rem' }}>
-                    {isMultipleRelationships ? 'RELATIONSHIPS:' : 'RELATIONSHIP:'}
-                  </span>
-                </Typography>
-                {detail.Relationships.map((relationship, idx) => (
-                  <Box key={idx} ml={2}>
-                    <RelationshipDetail relationship={relationship} />
-                  </Box>
-                ))}
-              </>
-            )}
-            {hasErrors &&
-              detail.Errors.map((error, idx) => (
-                <Box key={idx}>
-                  <UnsuccessfulEntityWithError modelName={modelName} error={error} />
-                </Box>
-              ))}
-          </Box>
-        );
-      })}
-    </>
-  );
+  ModelDetails: (value) => value && <ModelImportedSection modelDetails={value} />,
 };
 
-const ModelImportMessages = ({ message }) => (
-  <Typography>
-    <span style={{ fontWeight: 'bold', fontSize: '17px' }}>{`SUMMARY: `}</span>
-    <span style={{ fontSize: '17px' }}>{message}</span>
-  </Typography>
-);
-
-const DataToFileLink = ({ data }) => {
-  // convert the trace to a file
-  const dataString = typeof data === 'string' ? data : JSON.stringify(data, null, 2);
-  const file = new File([dataString], 'trace.txt', { type: 'text/plain' });
-
-  return (
-    <TitleLink href={URL.createObjectURL(file)} download="trace.txt">
-      Download Trace
-    </TitleLink>
-  );
+const EventTypeFormatters = {
```

Comment:
we have a defined mapping for formatters specific to events , nice  :clap:

Reply:
@FaheemOnHub FYI